### PR TITLE
SONARKT-86 Update rules metadata

### PR DESCRIPTION
**Pull request by margarita-nedzelska-sonarsource**

### CODE REVIEW

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1172.json [hunk 1]
```diff
@@ -12,5 +12,10 @@
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-1172",
   "sqKey": "S1172",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC12-C."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1172.json:18]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S126.json [hunk 2]
```diff
@@ -6,11 +6,15 @@
...
+  "tags": [],
   "defaultSeverity": "Critical",
   "ruleSpecification": "RSPEC-126",
   "sqKey": "S126",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC01-C.",
+      "MSC57-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S126.json:17]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1313.json [hunk 3]
```diff
@@ -16,6 +16,9 @@
   "securityStandards": {
     "OWASP": [
       "A3"
+    ],
+    "CERT": [
+      "MSC03-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1313.json:21]: Unrlated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1656.json [hunk 4]
```diff
@@ -6,11 +6,14 @@
...
-  ],
+  "tags": [],
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-1656",
   "sqKey": "S1656",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC12-C."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1656.json:16]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1763.json [hunk 5]
```diff
@@ -17,6 +17,10 @@
   "securityStandards": {
     "CWE": [
       561
+    ],
+    "CERT": [
+      "MSC12-C.",
+      "MSC56-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1763.json:23]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1764.json [hunk 6]
```diff
@@ -6,11 +6,14 @@
...
-  ],
+  "tags": [],
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-1764",
   "sqKey": "S1764",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC12-C."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1764.json:16]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1862.json [hunk 7]
```diff
@@ -13,5 +13,10 @@
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-1862",
   "sqKey": "S1862",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC12-C."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1862.json:19]: unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S2068.json [hunk 8]
```diff
@@ -22,6 +22,9 @@
     ],
     "OWASP": [
       "A2"
+    ],
+    "CERT": [
+      "MSC03-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S2068.json:27]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S2245.json [hunk 9]
```diff
@@ -18,10 +19,22 @@
...
+    "OWASP Mobile": [
+      "M5"
+    ],
+    "MASVS": [
+      "MSTG-CRYPTO-6"
+    ],
+    "CERT": [
+      "MSC02-J.",
+      "MSC30-C.",
+      "MSC50-CPP."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S2245.json:37]: Unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S4830.json [hunk 10]
```diff
@@ -24,6 +25,15 @@
...
       "A3"
+    ],
+    "OWASP Mobile": [
+      "M3"
+    ],
+    "MASVS": [
+      "MSTG-NETWORK-3"
+    ],
+    "CERT": [
+      "MSC61-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S4830.json:36]: unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S5542.json [hunk 11]
```diff
@@ -25,6 +26,15 @@
...
       "A3"
+    ],
+    "OWASP Mobile": [
+      "M5"
+    ],
+    "MASVS": [
+      "MSTG-CRYPTO-3"
+    ],
+    "CERT": [
+      "MSC61-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S5542.json:37]: unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S5547.json [hunk 12]
```diff
@@ -25,6 +26,15 @@
...
       "A6"
+    ],
+    "OWASP Mobile": [
+      "M5"
+    ],
+    "MASVS": [
+      "MSTG-CRYPTO-3"
+    ],
+    "CERT": [
+      "MSC61-J."
```
Comment (margarita-nedzelska-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S5547.json:37]: unrelated

File: sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S3329.json [hunk 13]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Cipher Block Chaining IV's should be unpredictable",
+  "title": "Cipher Block Chaining IV\u0027s should be unpredictable",
```
Comment (johann-beleites-sonarsource) [inline sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S3329.json:2]: This title is actually interesting in that:
1. it's using a different char for `'`
2. It shouldn't be using `'` at all, as the `s` in `IVs` here is plural and not possessive. 
See https://github.com/SonarSource/rspec/pull/246

**Review by johann-beleites-sonarsource — APPROVED** [review]
Looks good, thanks for updating. I found one tiny mistake while passing over some of the titles and opened a corresponding PR. If you agree, we could update the rspec before merging.
